packages/fluentui/react/src/components/Dropdown/Dropdown.tsx

```diff
@@ -82,6 +82,9 @@ export interface DropdownProps extends UIComponentProps<DropdownProps, DropdownS
   /** The initial value (or value array if the array has multiple selection). */
   defaultValue?: ShorthandValue<DropdownItemProps> | ShorthandCollection<DropdownItemProps>;
 
+  /** A Dropdown can be disabled. */
+  disabled?: boolean;
```

Comment:
I would use some description consistent to the Button's disabled prop: 
`/** A button can show that it cannot be interacted with. */`